tasks/90_inventory-update/inventory-update.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * @description
+ * Compares and updates the inventory stored in a 2D array against a second
+ * 2D array of a fresh delivery. The returned inventory array
+ * is in alphabetical order by item.
+ * @example
+ * // returns [[42, "Bowling Ball"], [2, "Dirty Sock"]
+ * updateInventory([[21, "Bowling Ball"], [2, "Dirty Sock"]],
+ * [[21, "Bowling Ball"]],)
+ * @param {Array} currentInv
+ * @param {Array} newInv
+ * @return {Array}
+ */
+export function updateInventory(currentInv, newInv) {
+  const resultInv = currentInv;
```

Comment:
For what do you needs this?
If you wanna reassign array,  please use Array.from

Reply:
i used the new array to prevent mutation of the one, that comes as parameter.